propka/lib.py

```diff
@@ -149,68 +236,17 @@ def loadOptions(*args):
     parser = OptionParser(usage)
```

Comment:
`OptionParser` import and instance can be removed

Reply:
Addressed in 6c67ef152c1f7914f84593c53d9435c1e11790f9.